File: ex11.py
def divisors(num):
    listd = []
    for i in range(1,num):
        if num%i==0:
            listd.append(i)
    sum=0
    for w in listd:
        sum=w+sum
    if sum==num:
        print("the number is perfect")
    else:
        print("the number is not perfect")
print(divisors(6))

File: ex11.c
#include <stdio.h>

int perfectnum (int number){
	int i, sum=0;
	for (i=number-1; i>=1; i--){
		if (number%i==0){
			sum=sum+i;
		}
	}
	if (sum==number){
		printf("the number is perfect");
	}
	else {
		printf("the number is not perfect");
	}
}

int main (void){
	perfectnum(6);
	return 0;
}
